ignite/contrib/handlers/throughput_benchmark_wrapper.py

```diff
@@ -0,0 +1,216 @@
+import torch
+import torch.jit
+import contextlib
+from typing import Callable, Union
+from ignite.engine import Events, Engine
+from torch.utils import ThroughputBenchmark
+from torch.utils.throughput_benchmark import ExecutionStats  # for linting
+
+
+# TODO: Discuss device implications
+class ThroughputBenchmarkWrapper:
+    """
+    This class is wrapper around the ThroughputBenchmark wrapper from torch.
+    ThroughputBenchmark is responsible for executing a PyTorch
+    module (nn.Module or ScriptModule)
+    under an inference server like load. It can emulate multiple calling threads
+    to a single module provided.
+
+    Please note that even though nn.Module is supported, it might incur an overhead
+    from the need to hold GIL every time we execute Python code or pass around
+    inputs as Python objects. As soon as you have a ScriptModule version of your
+    model for inference deployment it is better to switch to using it in this
+    benchmark.
+
+    Args:
+        model (Union[torch.nn.Module, torch.jit.ScriptModule]): model which will
```

Comment:
just torch.nn.Module should be sufficient as a type I guess. I think, a ScriptModule is a subclass of the usual model, IIRC

Reply:
Yes, you are correct. I will change it in my next commit. :)